include/experimental/__p0009_bits/layout_left.hpp

```diff
@@ -128,6 +128,26 @@ struct layout_left {
       :__extents(other.extents())
     { }
 
+    MDSPAN_TEMPLATE_REQUIRES(
+      class OtherMapping,
+      /* requires */ (
+        _MDSPAN_TRAIT(is_same, typename OtherMapping::layout_type, layout_stride) &&
+        _MDSPAN_TRAIT(is_same, typename OtherMapping::layout_type::template mapping<typename OtherMapping::extents_type>, OtherMapping) 
+      )
+    )
+    MDSPAN_CONDITIONAL_EXPLICIT((Extents::rank()!=0)) // needs two () due to comma
+    MDSPAN_INLINE_FUNCTION _MDSPAN_CONSTEXPR_14
+    mapping(OtherMapping const& other) // NOLINT(google-explicit-constructor)
+      :__extents(other.extents())
+    {
+       size_t stride = 1;
+       for(size_type r=0; r<__extents.rank(); r++) {
+         if(stride != other.stride(r))
+           throw(std::runtime_error("Assigning layout_stride to layout_left with invalid strides."));
```

Comment:
```suggestion
           throw std::runtime_error("Assigning layout_stride to layout_left with invalid strides.");
```
also noting they were no other occurrence of throwing on precondition violation in the current implementation.